.github/workflows/spec-update.yml

```
@@ -55,9 +55,11 @@ jobs:
           path: node_modules/@aws-cdk/aws-service-spec/db.json.gz
           if-no-files-found: error
 
-      # Build @aws-cdk/spec2cdk and run L1 gen script to generate base files for new modules
+      # Build @aws-cdk/spec2cdk and @aws-cdk/mixins-preview, then run L1 gen script to generate base files for new modules
       - name: Build @aws-cdk/spec2cdk
         run: lerna run build --stream --no-progress --skip-nx-cache --scope @aws-cdk/spec2cdk
+      - name: Build @aws-cdk/mixins-preview
+        run: lerna run build --stream --no-progress --skip-nx-cache --scope @aws-cdk/mixins-preview
       - name: Generate L1s
         working-directory: packages/aws-cdk-lib
         run: yarn gen
```

I think this should do

```suggestion
      - name: Generate code
        run: yarn gen
        run: lerna run gen --stream --no-progress --skip-nx-cache --scope aws-cdk-lib --scope @aws-cdk/mixins-preview
```